src/domains/citizen-auth-common/99_locals.tf

```diff
@@ -33,4 +33,16 @@ locals {
 
   vnet_common_name_itn                = "${local.common_project_itn}-common-vnet-01"
   vnet_common_resource_group_name_itn = "${local.common_project_itn}-common-rg-01"
+
+  location           = "westeurope"
+  secondary_location = "italynorth"
```

Comment:
We already have a local containing the italynorth location. It's called itn_location on row 30
```suggestion
```